{"uuid": "65ff7e2e-698f-f289-cbbc-00000000000b", "counter": 15, "stdout": "\r\nTASK [cloud-init-kvm : Check if debian base image already exists] **************", "start_line": 38, "end_line": 40, "runner_ident": "create_vm_20210307130914", "event": "playbook_on_task_start", "pid": 7849, "created": "2021-03-07T13:09:18.576254", "parent_uuid": "65ff7e2e-698f-f289-cbbc-000000000006", "event_data": {"playbook": "kvm_cloud_init.yml", "playbook_uuid": "0d23abb5-bee0-4714-95d7-e9dbff11d2af", "play": "kvm", "play_uuid": "65ff7e2e-698f-f289-cbbc-000000000006", "play_pattern": "kvm", "task": "Check if debian base image already exists", "task_uuid": "65ff7e2e-698f-f289-cbbc-00000000000b", "task_action": "stat", "task_args": "", "task_path": "/home/diogo/Documents/ansibleAPP/ansible_data_dir/kvm/project/roles/cloud-init-kvm/tasks/main.yml:4", "role": "cloud-init-kvm", "name": "cloud-init-kvm : Check if debian base image already exists", "is_conditional": false, "uuid": "65ff7e2e-698f-f289-cbbc-00000000000b"}}